dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/logs_view_config_panel_item.tsx

```diff
@@ -170,14 +170,15 @@ export const LogsViewConfigPanelItem = ({
 
   const getLogsViewUI = () => {
     const list = configList[GROUPBY] ? configList[GROUPBY] : [];
+    const disabledColor = "#fafbfd";
```

Comment:
Can we move this to the constant file?

Reply:
yes sure. We will move this to constant file.

Reply:
Done!